### PR TITLE
Init coordinator setup

### LINKED ISSUE #10: Setup coordinator app
This will contain our 10101 master node, i.e. a node where every 10101 user connects to.

Target platform should be linux as we intent to run it in a docker container.

Note: #9 should be in the same repository, i.e. in https://github.com/get10101/10101

### PR DESCRIPTION
**Pull request by bonomat**
CI actions tested on my fork and docker container works.

resolves #10

### CODE REVIEW

File: coordinator/.gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
+Cargo.lock
```
Comment (holzeis) [inline coordinator/.gitignore:8]: Based on the link that you have added.
> If you’re building a non-end product, such as a rust library that other rust [packages](https://doc.rust-lang.org/cargo/appendix/glossary.html#package) will depend on, put Cargo.lock in your .gitignore. If you’re building an end product, which are executable like command-line tool or an application, or a system library with crate-type of staticlib or cdylib, check Cargo.lock into git.

The coordinator is the later one "end product, which are executable like command-line tool or an application,..", hence the Cargo.lock **should not** be added to the .gitignore.

File: coordinator/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("Hello, world!");
```
Comment (holzeis) [inline coordinator/src/main.rs:2]: 👋

**Review by holzeis — APPROVED** [review]

**Comment by bonomat** [comment]
bors ping

**Comment by bonomat** [comment]
bors ping

**Comment by bonomat** [comment]
Going ahead and merging this, once merged, bors should be available :)

**Pull request merged**